apps/privacy-panel/test/unit/app_list_test.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+
+var realMozApps;
+var realMozL10n;
+var fakeApp1;
+var fakeApp2;
+var fakeApp3;
+
+suite('AppList', function() {
+
+  suiteSetup(function(done) {
+    var apps = [
+      {
+        manifest: {
+          name: 'Mozilla Fake App 1',
+          launch_path: '/fakeapp1/index.html',
+          type: 'privileged',
+          permissions: {
+            permission_1: {}
+          },
+          developer: {
+            name: 'Flying Platypus',
+            url: 'https://flying.platypus.com/'
+          },
+          icons: {
+            84: '/style/icons/settings_84.png',
+            126: '/style/icons/settings_126.png',
+            142: 'style/icons/settings_142.png',
+            189: '/style/icons/settings_189.png',
+            284: '/style/icons/settings_284.png'
+          }
+        },
+        manifestURL: 'http://fakeapp1/manifest.webapp'
+      },
+      {
+        manifest: {
+          name: 'Mozilla Fake App 2',
+          launch_path: '/fakeapp2/index.html',
+          type: 'privileged',
+          developer: {
+            name: 'Mighty Duck',
+            url: 'https://mighty.duck.com/'
+          },
+          permissions: {
+            permission_2: {}
+          }
+        },
+        manifestURL: 'http://fakeapp2/manifest.webapp'
+      },
+      {
+        manifest: {
+          name: 'Mozilla Fake App 3',
+          launch_path: '/fakeapp3/index.html',
+          type: 'web',
+          developer: {
+            name: 'Mighty Duck',
+            url: 'https://mighty.duck.com/'
+          },
+          permissions: {
+            permission_2: {}
+          }
+        }
+      }
+    ];
+
+    require([
+      'mocks/mock_navigator_moz_apps',
+      'mocks/mock_l10n'
+    ],
+    function(mozApps, mozL10n) {
+      realMozApps = navigator.mozApps;
+      navigator.mozApps = mozApps;
+      navigator.mozApps.mApps = apps;
+
+      realMozL10n = navigator.mozL10n;
+      navigator.mozL10n = mozL10n;
```

Comment:
yes, because `js/app_list.js` uses mozL10n to sort permissions by localized name.